cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -343,31 +343,38 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             event.preventDefault();
             this.$('#id_time_limit_div').hide();
             this.$('.exam-review-rules-list-fields').hide();
+            this.$('.timed-exam-options-list-fields').hide();
             this.$('#id_time_limit').val('00:00');
         },
-        selectSpecialExam: function (showRulesField) {
+        selectSpecialExam: function (additionalOptions) {
             this.$('#id_time_limit_div').show();
             if (!this.isValidTimeLimit(this.$('#id_time_limit').val())) {
                 this.$('#id_time_limit').val('00:30');
             }
-            if (showRulesField) {
+            if ($.inArray("review-rules", additionalOptions) !== -1) {
                 this.$('.exam-review-rules-list-fields').show();
             }
             else {
                 this.$('.exam-review-rules-list-fields').hide();
             }
+            if ($.inArray("hide-after-due", additionalOptions) !== -1) {
+                this.$('.timed-exam-options-list-fields').show();
+            }
+            else {
+                this.$('.timed-exam-options-list-fields').hide();
+            }
         },
         setTimedExam: function (event) {
             event.preventDefault();
-            this.selectSpecialExam(false);
+            this.selectSpecialExam(["hide-after-due"]);
```

Comment:
This is a special case of the method commented on above - it's saying "make the `hide-after-due` field visible", which should always happen for timed exams.